scripts/stubtest/allowlist_todo.txt

```diff
@@ -46,28 +46,20 @@ rest_framework.fields.DateField.to_internal_value
 rest_framework.fields.DateTimeField.__init__
 rest_framework.fields.DateTimeField.to_internal_value
 rest_framework.fields.DecimalField.__init__
-rest_framework.fields.DictField.__init__
 rest_framework.fields.DictField.initial
 rest_framework.fields.DurationField.to_internal_value
 rest_framework.fields.Field.__init__
 rest_framework.fields.FilePathField.__init__
```

Comment:
Note: some `XxxField.__init__` methods remain because argument types aren't precise. But I'd rather fix that separately.